### PR TITLE
Melkehitys-1023

### PR DESCRIPTION
**Pull request by natlibfi-jonollil**
### Updated:
Package

### Added
Logging

### Local test settings:
- melinda-record-import-api #melhehitys-1020 build locally
- quay.io/natlibfi/melinda-record-import-controller:test
- melinda-record-import-importer-dummy #melkehitys-1023 build locally
- melinda-record-import-transformer-helmet #melkehitys-1020 build locally
 and melinfa-record-import-transformer-dummy #melkehitys-1023 build locally

### CODE REVIEW

**Comment by natlibfi-jonollil** [comment]
Docker error
unauthorized: access to the requested resource is not authorized

File: package.json [hunk 1]
```diff
@@ -23,27 +23,29 @@
...
+		"@babel/register": "^7.6.2",
 		"babel-eslint": "^10.0.1",
-		"cross-env": "^5.2.0",
-		"eslint": "^5.5.0",
-		"eslint-config-xo": "^0.26.0",
-		"eslint-plugin-import": "^2.13.0",
-		"nodemon": "^1.18.4",
+		"cross-env": "^6.0.0",
+		"eslint": "^6.4.0",
+		"eslint-config-xo": "^0.27.1",
```
Comment (ghost) [inline package.json]: Should be `>=0.27.1 <1.0.0`

File: src/import.js [hunk 2]
```diff
@@ -27,9 +27,20 @@
...
+const {createLogger} = Utils;
+
 export default function () {
-	return async () => {
-		const id = new Date().getTime();
-		return {id, status: 'CREATED'};
+	const Logger = createLogger();
+
+	return async message => {
+		Logger.log('debug', 'Dummy importer! Dosent import records to Melinda');
```
Comment (ghost) [inline src/import.js]: Fix typo: `Dosent` -> `Doesn't`

**Review by ghost — CHANGES_REQUESTED** [review]
About the Docker unauthorized error: This repository had PRs enabled in Drone which caused the Docker-build stage (Builds triggered from PRs cannot access secrets by default).

**Comment by ghost** [comment]
Great! Will merge when API, commons, controller and Helmet-transformer are ready.

**Review by ghost — CHANGES_REQUESTED** [review]
[@natlibfi/melinda-record-import-commons](https://www.npmjs.com/package/@natlibfi/melinda-record-import-commons/v/7.0.0) has been published. Update dependency to point to that version in NPM.

**Review by ghost — APPROVED** [review]

**Pull request merged**